diagnostic.md

```diff
@@ -48,11 +50,11 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    Add it to the model? I dont really understand the question.
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+    {{#each model as |something|}}
```

Comment:
Yup, the important part here is the access to `model`. How you parse that in HTMLBars is up to you, but the `model` is what's providing the data to a template. 1/2 credit